src/pages/list/index.jsx

```diff
@@ -1,11 +1,13 @@
 import React from "react";
 import Header from "../../components/Header";
+import ListItem from "../../components/ListItem";
+import places from "../../data/places";
 
 export default function ListPage() {
   return (
     <>
       <Header />
-      <div>List</div>
+      <ul>{places.map((place) => ListItem(place))}</ul>
```

Comment:
```suggestion
      <ul>{places.map((place) => (<ListItem place={place} />))}</ul>
```

컴포넌트는 컴포넌트답게 표시하는 게 좋지 않을까요.

그 전에 우선 ListItem `함수`를 `컴포넌트`로 바꾸시고요